lib/reporting/protocols_report.rb

```diff
@@ -288,6 +294,38 @@ def loa_issuers_query
       QUERY
     end
 
+    def aal3_issuers_data
+      @aal3_issuers_data ||= begin
+        cloudwatch_client.fetch(
+          query: aal3_issuers_query,
+          from: time_range.begin,
+          to: time_range.end,
+        ).
+          map { |slice| slice['issuer'] }.
+          uniq
+      end
```

Comment:
no need for `begin...end` syntax
```suggestion
      @aal3_issuers_data ||= cloudwatch_client.fetch(
        query: aal3_issuers_query,
        from: time_range.begin,
        to: time_range.end,
      ).
        map { |slice| slice['issuer'] }.
        uniq
```